ehcache-impl/src/main/java/org/ehcache/config/builders/CacheManagerBuilder.java

```diff
@@ -242,7 +242,11 @@ public CacheManagerBuilder<T> withDefaultSizeOfMaxObjectGraph(long size) {
    * @param size the max object size
    * @param unit the max object size unit
    * @return a new builder with the added configuration
+   *
+   * @deprecated Use {@link #withDefaultSizeOfMaxObjectGraph(long) } instead
```

Comment:
Again, not clear why this method is a good substitute? In fact it's not clear why this method isn't also being deprecated. The whole size-of thing and everything derived from it is being deprecated for removal. There is no  substitute for it.

Reply:
removed the @deprecated from java doc